lib/static/css/main.css

```diff
@@ -523,13 +523,12 @@ header h2 {
   font-size: 16px;
 }
 @media (max-width: 375px) {
-  .logo {
-    display: none;
-  }
-
   .headerTitle {
     font-size: 17px;
   }
+  .headerTitleHidden {
+    display: none !important;
```

Comment:
Is `!important` needed?

Reply:
`headerTitleHidden` is not a good name because it's only hidden on smaller widths. How about `headerTitleWithLogo`?

Reply:
Yes, `!important` is needed here.

Renamed to `headerTitleWithLogo`.